packages/datadog-plugin-azure-functions/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+const {
+  FakeAgent,
+  hookFile,
+  createSandbox,
+  curlAndAssertMessage
+} = require('../../../../integration-tests/helpers')
+const { spawn } = require('child_process')
+const { assert } = require('chai')
+const findProcess = require('find-process')
+const waitOn = require('wait-on')
+
+describe('esm', () => {
+  let agent
+  let proc
+  let sandbox
+
+  withVersions('azure-functions', '@azure/functions', version => {
+    before(async function () {
+      this.timeout(50000)
+      sandbox = await createSandbox([`@azure/functions@${version}`, 'azure-functions-core-tools@4'], false,
+        ['./packages/datadog-plugin-azure-functions/test/integration-test/fixtures/*'])
+    })
+
+    after(async function () {
+      this.timeout(50000)
+      await sandbox.remove()
+    })
+
+    beforeEach(async () => {
+      agent = await new FakeAgent().start()
+    })
+
+    afterEach(async () => {
+      const azureFuncProc = await findProcess('name', 'func', true)
+      const azureFuncProcPid = azureFuncProc[0]?.pid ?? null
+      azureFuncProcPid !== null && process.kill(azureFuncProcPid, 'SIGKILL')
```

Comment:
Could the process exit gracefully instead?

Reply:
I'm not sure why but the Azure Function process doesn't stop when using a `SIGTERM` signal. It only stops with the `SIGKILL` signal.

Reply:
What I meant is to not need to send it a signal and let it exit gracefully, although I don't know if it's meant to be able to do that. If it's not then this is fine.